### PR TITLE
Enhancement/pagination and serialize

### PR DESCRIPTION
**Pull request by mayanksyook**
Pagination fix and added prop to disable serial no column

### CODE REVIEW

File: src/containers/table/index.js [hunk 1]
```diff
@@ -152,16 +152,21 @@ class TableComponent extends Component {
                                                   this.enableBulkSelect({ checked }, filterProps.data)
                                                 }
                                               />
-                                            ) : null}
+                                            </div>
+                                          </Table.HeaderCell>
+                                        ) : null}
+                                        {this.props.serialize && (
```
Comment (nithil) [inline src/containers/table/index.js:158]: this is a boolean field, rename accordingly. isShowSerialNumber

update readme regarding this prop

File: src/components/pagination/index.js [hunk 2]
```diff
@@ -16,61 +16,58 @@ const Pagination = props => {
...
-                <Select
-                  className="pagination_select"
-                  value={props.rowsPerPage}
-                  options={pageOptions}
-                  onChange={props.onSelectRowsPerPage}
-                  isClearable={false}
-                  isSearchable={false}
-                  menuPlacement="auto"
-                  menuPortalTarget={document.querySelector('#root')}
+    <div className="paginationFooter" colSpan={props.numberOfColumns}>
```
Comment (anees-syook) [inline src/components/pagination/index.js:19]: can we remove the `colSpan` also?

File: src/containers/table/index.js [hunk 3]
```diff
@@ -183,15 +188,15 @@ class TableComponent extends Component {
...
-                                              <div
-                                                style={{
-                                                  display: 'flex',
-                                                  justifyContent: 'space-between',
-                                                  flexDirection: props.showStatusIcon ? 'row-reverse' : null,
-                                                  alignItems: 'baseline',
-                                                }}>
-                                                {hasBulkActions && includeCheckbox !== false ? (
+                                            {hasBulkActions && includeCheckbox !== false ? (
+                                              <Table.Cell>
```
Comment (anees-syook) [inline src/containers/table/index.js:192]: can we use just `includeCheckbox` here?

File: src/index.css [hunk 4]
```diff
@@ -33,6 +33,10 @@ code {
   border-bottom: 1px solid rgb(241, 245, 247) !important;
 }
 
+.ui.ribbon.label {
+  left: 0px !important;
```
Comment (anees-syook) [inline src/index.css:37]: I think this will have an effect if we need the default ribbon else where. Let add a custom className also?

**Pull request merged**